acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/multitenancy/MultiTenancyAcceptanceTest.java

```diff
@@ -0,0 +1,148 @@
+
+    // privateFrom value from above transaction
+    final String privateFrom = "A1aVtMxLCUHmBVHXoZzzBgPbW/wj5axDpW9X8l91SGo=";
+
+    // Create a transaction
+    final Account sender = accounts.createAccount("account1");
+    final Hash transactionHash = node.execute(accountTransactions.createTransfer(sender, 50));
+    multiTenancyCluster.verify(sender.balanceEquals(50));
+
+    final ObjectMapper mapper = new ObjectMapper();
+    final String receiveResponse =
+        mapper.writeValueAsString(
+            new ReceiveResponse(
+                base64SignedPrivateTransactionRLP.getBytes(UTF_8), privacyGroupId, "senderKey"));
+
+    stubFor(post("/receive").willReturn(ok(receiveResponse)));
+
+    node.verify(priv.privGetPrivateTransactionSuccess(transactionHash.toString(), privateFrom));
+  }
+
+  @Test
+  public void shouldCreatePrivacyGroup() throws JsonProcessingException {
+    final ObjectMapper mapper = new ObjectMapper();
+    final String groupId = "groupId";
+    final String createGroupResponse = mapper.writeValueAsString(testPrivacyGroup());
+
+    CreatePrivacyGroupParameter params =
+        new CreatePrivacyGroupParameter(
+            List.of(key1, key2, key3), "GroupName", "Group description.");
+    stubFor(post("/createPrivacyGroup").willReturn(ok(createGroupResponse)));
+
+    node.useAuthenticationTokenInHeaderForJsonRpc(token);
+    node.verify(priv.privCreatePrivacyGroupSuccess(params, groupId));
+  }
+
+  @Test
+  public void shouldDeletePrivacyGroup() throws JsonProcessingException {
+    final ObjectMapper mapper = new ObjectMapper();
+    final String retrieveGroupResponse = mapper.writeValueAsString(testPrivacyGroup());
+    stubFor(post("/retrievePrivacyGroup").willReturn(ok(retrieveGroupResponse)));
+
+    final String deleteGroupResponse = mapper.writeValueAsString(privacyGroupId);
+    stubFor(post("/deletePrivacyGroup").willReturn(ok(deleteGroupResponse)));
+
+    node.useAuthenticationTokenInHeaderForJsonRpc(token);
+    node.verify(priv.privDeletePrivacyGroupSuccess(privacyGroupId));
+  }
+
+  @Test
+  public void shouldFindPrivacyGroup() throws JsonProcessingException {
+    final ObjectMapper mapper = new ObjectMapper();
+    final String findGroupResponse = mapper.writeValueAsString(List.of(testPrivacyGroup()));
+    stubFor(post("/findPrivacyGroup").willReturn(ok(findGroupResponse)));
+
+    final String[] paramArray = {enclaveKey};
+
+    node.useAuthenticationTokenInHeaderForJsonRpc(token);
+    node.verify(priv.privFindPrivacyGroupSuccess(paramArray));
+  }
+
+  private PrivacyGroup testPrivacyGroup() {
+    return new PrivacyGroup(
+        privacyGroupId, PrivacyGroup.Type.PANTHEON, "test", "testGroup", List.of(enclaveKey));
```

Comment:
oh we still have that ... didn't we end up creating a duplicate of PANTHEON -> BESU?

Reply:
Doesn't seem like it.  Will update.